src/utils.tsx

```diff
@@ -188,7 +196,12 @@ export const renderText = <Us extends DefaultUserType<Us> = DefaultUserType>(
     if (noParsingNeeded.length > 0 || linkIsInBlock) return;
 
     const displayLink =
-      type === 'email' ? value : truncate(value.replace(detectHttp, ''), 20);
+      type === 'email'
```

Comment:
Maybe we could move the truncation logic inside the custom renderer for `link` tags. That way we don't need to have a separate option for `truncationLimit`, but instead users can provide a custom renderer for `link` that truncates in some different way.